### ISSUE TITLE
Python 3.6/nightly: ValueError: cannot use LOCALE flag with a str pattern

### ISSUE THREAD

**Issue by adamcharnock** [kennethreitz/tablib#254]
The following error seems to be raised when testing my project against Python nightly builds.

I appreciate you may not support this right now, but thought you may appreciate knowing about this in advance of any release.

I'm going to guess that this would be fixed by changing changing the pattern in [ExcelFormulaLexer.py:51](https://github.com/kennethreitz/tablib/blob/master/tablib/packages/xlwt3/ExcelFormulaLexer.py#L51) to be `r'...'` rather than `'...'`

```
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/setup.py", line 7, in <module>
    name='swiftwind',
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/__init__.py", line 3, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/core.py", line 15, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/formats/__init__.py", line 6, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/formats/_csv.py", line 6, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/compat.py", line 25, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/packages/xlwt3/__init__.py", line 5, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/packages/xlwt3/Row.py", line 7, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/packages/xlwt3/ExcelFormula.py", line 1, in <module>
  File "/tmp/easy_install-9lyndnho/tablib-0.11.2/tablib/packages/xlwt3/ExcelFormulaLexer.py", line 52, in <module>
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/re.py", line 232, in compile
    return _compile(pattern, flags)
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/sre_parse.py", line 861, in parse
    p.pattern.flags = fix_flags(str, p.pattern.flags)
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/sre_parse.py", line 828, in fix_flags
    raise ValueError("cannot use LOCALE flag with a str pattern")
ValueError: cannot use LOCALE flag with a str pattern
```

**Comment by andriisoldatenko** [comment]
Please review my PR https://github.com/kennethreitz/tablib/pull/263

**Comment by gerhardkmn** [comment]
Am Sun, 18 Dec 2016 06:56:39 -0800
schrieb Andrii Soldatenko <notifications@github.com>:

> Please review my PR https://github.com/kennethreitz/tablib/pull/263
> 

I've just approved the two changes.

I tried them with xlwt-1.1.2 and tablib-0.11.3

and no error messages appear.

Also I tried a self written application which uses pyexcel which
internally calls xlwt and my application now works like a charm with
Python 3.6.0rc2

Gerhard Kirchmann

**Comment by iurisilvio** [comment]
#263 was merged, so I'm closing this issue.

**Comment by NarutoLiao** [comment]
I've met the same problem with you. Have you solved it?How did you deal with it?
My problem occurs when I import xlwt.

**Comment by benf-wspdigital** [comment]
@iurisilvio, thanks for merging the fix.

There does not yet appear a release later than 0.11.4, where can we get a release that fixes this issue for Python 3.6?

**Comment by MrBrownWins** [comment]
Updating to tablib==0.11.5 worked for me, python 3.6.1
